src/utils/dateRange.ts

```diff
@@ -35,10 +36,14 @@ export function getNoDataForDateRangeString(key: string = 'no_data_for_date', of
   const endDate = format(today, 'd');
   const startDate = format(startOfMonth(today), 'd');
 
-  return intlHelper(intl.formatMessage(messages.NoDataForDate, { count: getDate(today), startDate, endDate, month }));
+  return intlHelper(intl.formatMessage(key, { count: getDate(today), startDate, endDate, month }));
 }
 
-export function getForDateRangeString(value: string | number, key: string = 'for_date', offset: number = 1) {
+export function getForDateRangeString(
+  value: string | number,
+  message: MessageDescriptor = messages.ForDate,
```

Comment:
Kept this as `key`? Or, perhaps we should change the other `key` params to match `message`?